tests/test_office_routes.py

```diff
@@ -12,34 +12,6 @@ def test_politicaloffice_creation (self):
         response = self.client.post('/api/v1/politicaloffices', json=self.politicaloffice)
         self.assertEqual(response.status_code, 201)
     
```

Comment:
I see you have deleted all other tests. Why is that so? I think they should pass too.